### ISSUE TITLE
Моды

### ISSUE THREAD

**Issue by 3adrot-Shot** [jxbc/slave-bot#1]
Почему я поставил мод который покупает челов где их стоимость закупается не выше максимальной в конфиге, но у меня к херам закупило челиков по миллиону, по два миллиона. Еще и в минусе остался дабы 8к в минуту!

**Comment by jxbc** [comment]
@3adrot-Shot , привет, убедись, что включен мод 2, так как мод 1 скупает страницы независимо от цены. Я проверил код и работоспособность, всё работает. Если мод все же работает не так, как ожидается, напиши пожалуйста в телеграм.

**Comment by 3adrot-Shot** [comment]
Косяк мой, поставил минимальное значение выше чем максимальное.
А будет обнова авто оков?

**Comment by jxbc** [comment]
@3adrot-Shot да, возможно сегодня-завтра сделаю полностью настраиваемые авто-оковы и напишу интуитивно понятный гайд, как все настраивать :)

**Comment by jxbc** [comment]
@3adrot-Shot обновил :)

**Comment by 3adrot-Shot** [comment]
> @3adrot-Shot обновил :)

Сяб, но у меня бот почему то перестал работать, просто ничу не выводит в консоль![Screenshot_20210330-115644.jpg](https://user-images.githubusercontent.com/62293559/112946801-618ce680-914f-11eb-95d0-dae694cda28f.jpg)![Screenshot_20210330-115623.jpg](https://user-images.githubusercontent.com/62293559/112946809-6356aa00-914f-11eb-8351-4d7102567910.jpg)

**Comment by 3adrot-Shot** [comment]
С каждым ожиданием у меня теряются рабы(

**Comment by jxbc** [comment]
@3adrot-Shot проверьте актуальность версии nodejs (должна быть установлена 15.12). Обновил бота! Извиняюсь за столь долгое ожидание...

**Comment by jxbc** [comment]
![image](https://user-images.githubusercontent.com/30753109/112966874-51770600-9153-11eb-9199-d3188a04908c.png)

Проверьте пожалуйста версию Node.js, у меня никаких ошибок нет, бот работает уже 10 минут

**Comment by 3adrot-Shot** [comment]
> ![image](https://user-images.githubusercontent.com/30753109/112966874-51770600-9153-11eb-9199-d3188a04908c.png)
> 
> Проверьте пожалуйста версию Node.js, у меня никаких ошибок нет, бот работает уже 10 минут

Это у вас какой мод включён?

**Comment by jxbc** [comment]
@3adrot-Shot второй мод, всё как в конфиге

**Comment by 3adrot-Shot** [comment]
> @3adrot-Shot второй мод, всё как в конфиге

Server Error, node я обновил
> Добавлено:
Изменил prod на свой и все заработало, в мануале про это не сказанно

**Comment by 3adrot-Shot** [comment]
> > @3adrot-Shot второй мод, всё как в конфиге
> 
> Server Error, node я обновил
> > Добавлено:
> Изменил prod на свой и все заработало, в мануале про это не сказано, и как то фильтр по максимальному значению неправильно робит, дабы он останавливает сканирование если нашёл максимальное число, хотя там дальше есть персонажи которые стоят дешевле после дорогого, ещё плохой минус что в массив чтобы указать одну работу нужно её дублировать 11 раз, сделай чтобы он узнавал максимальную длинну массива а уже потом рандомил, или как он у тебя там делает.
Ну и ещё сделать чтобы когда Славик проверялся он ещё и покупался, но ток при условиях Min Max.
И вроде при этих условиях все топ должно быть, ну ещё бы добавить контроллер в вк типо бота на группу чтобы можно было кого то атаковать, и следить, или чтобы какого то юзера постоянно контролить чтобы его купить. И кучу таких подобных функций

**Comment by 3adrot-Shot** [comment]
Хайп прошел, пляшем
